### DISCUSSION TITLE
Add Toman currency

### DISCUSSION THREAD

**Discussion by samyarkd** [medusajs/medusa#3850] [Feature Requests]
In Iran, we have two currencies one is only used by government and the other one used by people
The one that used by government is Rial and the other that is used by people is Toman
All the ecommerce websites use Toman too

The différance is that Toman has more value and it's shorter
For example :
Rial = ريال
Toman = تومان
10.000 ريال = 1.000 تومان
Toman one zero less that Rail

And the Payment service that I'm using is supporting both IRR and IRT but i prefer to use IRT

**Comment by einar-hjortdal** [comment]
I believe currencies are stored in `packages/admin-ui/ui/src/utils/currencies.ts` and `packages/medusa/src/utils/currencies.ts`. You could add it and submit a pull request.

**Reply by samyarkd** [reply]
Thanks you 👌

**Comment by samyarkd** [comment]
Hi 
sorry can anyone please review this PR? 
https://github.com/medusajs/medusa/pull/3916
@olivermrbl
